static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic.html',
```

Comment:
Is there any way you can combine all the contributions templates into one and pass everything in as variables? The reason I ask is that AB tests are bundled into standard.js which is the absolute bottom baseline of JS - so AB test should be kept as small and lean as possible and reducing the number of included modules is an important part of that.